web-ui/src/main/resources/catalog/style/gn_infolist.less

```diff
@@ -1,195 +1,223 @@
+        }
+        .abstract {
+          max-height: 1.5em !important;
+          overflow: hidden;
+          padding-left: 0;
+        }
+        .introduction {
+          display: none;
         }
       }
-      .header-card {
-        -moz-transform: translateY(0%);
-        -ms-transform: translateY(0%);
-        -webkit-transform: translateY(0%);
-        margin: 1%;
-        transform: translateY(0%);
+    }
+  }
+}
+// ----- large
+.gn-info-list-large {
+  .gn-info-list {
+    li {
+      width: 100%;
+      .resultcard {
+        .title {
+          padding-left: 0;
+        }
+        .content {
+          float: left;
+          max-width: 70%
+        }
+        .abstract {
+          max-height: auto !important;
+          max-height: 250px;
+          padding-left: 0;
+        }
+        .introduction {
+          margin-top: 10px;
+          .badge {
+            font-weight: normal;
+            background-color: #333;
+          }
+        }
+        .gn-md-thumbnail {
+          float: left;
+          display: block;
+          margin-top: 4px;
+          margin-right: 10px;
+        }
       }
-      .content-card {
-        color: @cardtext;
-        height:auto;
-        margin-bottom: 1%;
-        margin-top: 0%;
-        margin-top: 1%;
-        padding-right: 3%;
-        padding-top: 0%;
-        visibility:visible;
+      .resultcard.hasThumbnail {
+        &:hover {
+          .gn-img-thumbnail {
+            filter: none !important;
+          }
+        }
       }
-      .footer-card {
-        visibility: hidden;
+    }
+  }
+}
+// ----- blocks
+.gn-info-list-blocks {
+  .gn-info-list {
+    li {
+      width: 33%;
+      height: 250px;
+      .resultcard {
+        .title {
+          background: @gray-dark;
+          border: @gray-dark;
+          h4 {
+            color: #fff;
+            font-size: 15px;
+            font-weight: normal;
+            margin: 0;
+            max-width: 100%;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            line-height: 22px;
+          }
+        }
+        .abstract {
+          border: 1px solid #ccc;
+          height: 150px;
+          padding-bottom: 5px;
+        }
+        .introduction {
+          display: none;
+        }
+        .gn-md-thumbnail {
+          display: block;
+          width: 100%;
+        }
+        &:hover {
+          .title {
+            background: @gray-darker;
+          }
+        }
       }
-      .gn-md-thumbnail {
-        height: 0%;
-        left: 0%;
-        margin: 0;
-        position: absolute;
-        top: 0%;
-        width: 5%;
-        z-index: 100;
-        img {
-          max-height: 0%;
-          max-width: 0%;
+      .resultcard.hasThumbnail {
+        &:hover {
+          .gn-img-thumbnail {
+            filter: none !important;
+          }
         }
       }
+      .abstract {
+        display: none !important;
+      }
+    }
+    li:hover {
```

Comment:
I think it's unintuitive that some hover effects are on the `li` container, and some are on the `section.resultcard` element. This means that sometimes the card is zoomed in but still in b&w. If possible, all hover effects should be on the same container.